.eslintrc.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
         "jest/no-focused-tests": "error",
         "jest/no-identical-title": "error",
         "jest/prefer-to-have-length": "warn",
-        "jest/valid-expect": "error"
+        "jest/valid-expect": "error",
+        "react":"",
```

Comment:
Why are there two `rules` properties? Also, what does the `react` rule do there?

Reply:
You still haven't answered my question here?